lib/config/default.js

```diff
@@ -1256,6 +1256,66 @@ defaultConfig.definition = () => ({
         default: ''
       }
     }
+  },
+  /**
+   * Security agent configurations
+   */
+  security: {
+    /**
+     * Enables/Disables security agent functions and generation of events.
+     */
+    enabled: {
+      formatter: boolean,
+      default: false
+    },
+
+    /**
+     * To completely disable security, set security.agent.enabled to false. This property is read only once at application start.
+     */
+    agent: {
+      enabled: {
+        formatter: boolean,
+        default: false
+      }
+    },
+
+    /**
+     *  NR security provides two modes IAST and RASP. Default is IAST
```

Comment:
Super-nitpicky, but I'd insert a colon and end with a period, like this: 

`NR security provides two modes: IAST and RASP. Default is IAST.`